package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ucirvine/eeeplus-theme",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
The version should be a minor increment since this is adding something new

Reply:
Okay, I'll change it. I did the same thing on the global navbar, I made the version 2.0.1. It should be 2.1.0. Should I change it? Thanks for sending the reminder about the versioning. I will look at it next time instead of relying on memory.

Reply:
Per standup chat, don't need the version but please fix the release name. See previous releases for examples